src/model_config_utils.cc

```diff
@@ -2247,9 +2247,34 @@ GetBackendType(const std::string& backend_name)
     return BackendType::BACKEND_TYPE_PYTORCH;
   }
 
+  if (backend_name == kPythonBackend) {
+    return BackendType::BACKEND_TYPE_PYTHON;
+  }
+
   return BackendType::BACKEND_TYPE_UNKNOWN;
 }
 
+bool
+ValidateBackendAndPlatformFields(const inference::ModelConfig& config)
+{
+  bool valid_pair = false;
+
+  auto backend_type = GetBackendType(config.backend());
```

Comment:
can we instead relax this logic to: "If GetBackendTypeFromPlaform() == UNKNOWN OR GetBackendTypeFromPlatform()==GetBackendType()"

In general it seems like we allow for the possibility that the platform is handled by the backend - so in cases where it isn't known - maybe that is up to the backend to validate.

Reply:
Fixed.